app/components/shared/Wrapping/Wrapper.jsx

```diff
@@ -127,7 +144,7 @@ export default class SecretWrapper extends Component {
                             anchorOrigin={{ "horizontal": "right", "vertical": "top" }}
                             targetOrigin={{"horizontal":"right","vertical":"bottom"}}
                         >
-                            <Menu onItemTouchTap={this.handleItemTouchTap}>
+                            <Menu onItemTouchTap={this.handleItemTouchTap} disableAutoFocus>
```

Comment:
this was required to stop the select menu from stealing the focus from the textfield on each onChange/onKeyDown event.